src/domain/journey/space/createSpace/plansTable/PlansTableDialog.tsx

```diff
@@ -41,13 +42,16 @@ const PlansTableDialog = ({ open, onClose, onSelectPlan }: PlansTableDialogProps
 
   const plansData = useMemo(
     () =>
-      (data?.platform.licensing.plans.filter(plan => plan.enabled).sort((a, b) => a.sortOrder - b.sortOrder) ?? []).map(
-        plan => ({
-          ...plan,
-          displayName: planTranslations[plan.name]?.displayName,
-          available: isPlanAvailable(plan),
-        })
-      ),
+      (
+        data?.platform.licensing.plans
+          .filter(plan => plan.enabled)
+          .filter(plan => plan.type === LicensePlanType.SpacePlan)
```

Comment:
Contextually filtering first by SpacePlan makes more sense, but it would make no difference to the result.